### PR TITLE
fix: downgrade fs-extra to v7 for fixing jest crashing

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

We can't upgrade to the latest version of `fs-extra` due to: https://github.com/jprichardson/node-fs-extra/issues/687

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [x] All new and existing tests are passing.

### CODE REVIEW

**Comment by clarkdo** [comment]
From the issue, I think issue is from `graceful-js`, should we wait ?

**Comment by pi0** [comment]
[vue-cli](https://github.com/vuejs/vue-cli/commit/43bdc333f4c714786bc8b5c67b34d74f465d8151) also downgraded because of this. Seems like a good workaround but the root cause is unlikely to be resolved anytime soon which means keep using an outdated dependency. I'm +1 to merge this fix but suggest using a simpler alternative for promise based fs.

**Comment by ricardogobbosouza** [comment]
@clarkdo Yeah, but it is causing problems in dev modules, for example sitemap-module

**Comment by clarkdo** [comment]
Cool, then please update yarn.lock and add it into ignore list in renovate config.

**Review by clarkdo — DISMISSED** [review]

File: packages/webpack/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "eventsource-polyfill": "^0.9.6",
     "extract-css-chunks-webpack-plugin": "^4.5.2",
     "file-loader": "^4.0.0",
-    "fs-extra": "^8.0.1",
+    "fs-extra": "^7.0.1",
```
Comment (clarkdo) [inline packages/webpack/package.json]: It seemed that webpack no longer depends on fs-extra,  this can be removed

Comment (ricardogobbosouza) [reply packages/webpack/package.json]: Yeah

File: renovate.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
     "dev"
   ],
   "ignoreDeps": [
-    "core-js"
+    "core-js",
+    "fs-extra"
```
Comment (clarkdo) [inline renovate.json]: @ricardogobbosouza Could you please add a todo comment here for tracking the real root issue ?

Comment (ricardogobbosouza) [reply renovate.json]: Sure!

**Comment by clarkdo** [comment]
@ricardogobbosouza Thanks 😄

**Comment by pi0** [comment]
Seems merged: https://github.com/isaacs/node-graceful-fs/pull/162

**Comment by clarkdo** [comment]
@pi0 @ricardogobbosouza PR: https://github.com/nuxt/nuxt.js/pull/5994, please verify and review

**Pull request merged**